vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -326,11 +328,13 @@
     <ClInclude Include="Views\Image\ReactImageBrush.h" />
     <ClInclude Include="Views\Impl\ScrollViewUWPImplementation.h" />
     <ClInclude Include="Views\Impl\SnapPointManagingContentControl.h" />
+    <ClInclude Include="Views\IXamlRootView.h" />
     <ClInclude Include="Views\KeyboardEventHandler.h" />
     <ClInclude Include="Views\PickerViewManager.h" />
     <ClInclude Include="Views\PopupViewManager.h" />
     <ClInclude Include="Views\RawTextViewManager.h" />
     <ClInclude Include="Views\ReactRootControl.h" />
+    <ClInclude Include="Views\ReactRootView2.h" />
```

Comment:
>2 [](start = 43, length = 1)

Any particular reason for being `2.h`  Should we use `ReactXamlRootView.h` and rename the class at the same time to better distinguish between:
```
class ReactRootView : public IXamlRootView {
```
and
```
struct ReactRootView : ReactRootViewT<ReactRootView> {
```

Reply:
Good point! I will do it. Though, I think we will remove this file completely at some point.